packages/twenty-server/src/engine/integrations/serverless/drivers/local.driver.ts

```diff
@@ -84,78 +103,87 @@ export class LocalDriver
     ${fileContent}
     `;
 
-    await fs.writeFile(tmpFilePath, modifiedContent);
+      await fs.writeFile(tmpFilePath, modifiedContent);
+
+      return await new Promise((resolve, reject) => {
+        const child = fork(tmpFilePath, { silent: true });
+
+        child.on('message', (message: object | ServerlessExecuteError) => {
+          const duration = Date.now() - startTime;
+
+          if ('errorType' in message) {
+            resolve({
+              data: null,
+              duration,
+              error: message,
+              status: ServerlessFunctionExecutionStatus.ERROR,
+            });
+          } else {
+            resolve({
+              data: message,
+              duration,
+              status: ServerlessFunctionExecutionStatus.SUCCESS,
+            });
+          }
+          child.kill();
+          fs.unlink(tmpFilePath);
```

Comment:
Be careful. This call returns a Promise, and you are not catching it here. If it throws, it will result in an unhandled error that might crash the process.

Depending on the importance of the file being deleted, we have two options:

1. Reject the overall promise if the deletion of the file fails
2. Discard the error that might happen

```suggestion
          child.kill();
          await fs.unlink(tmpFilePath);
          // or
          fs.unlink(tmpFilePath).catch(console.error)

          if ('errorType' in message) {
            resolve({
              data: null,
              duration,
              error: message,
              status: ServerlessFunctionExecutionStatus.ERROR,
            });
          } else {
            resolve({
              data: message,
              duration,
              status: ServerlessFunctionExecutionStatus.SUCCESS,
            });
          }
```

Reply:
The purpose of this try catch is only to catch the file not found error, so I will just move the try catch around the read file call only